### ISSUE TITLE
OBS crashes whenever I do anything dock-related

### ISSUE THREAD

**Issue by rodduarte666** [obsproject/obs-studio#11090]
### Operating System Info

Windows 11

### OBS Studio Version

30.2.2

### OBS Studio Log URL

https://obsproject.com/logs/2aHSasSyt9L2vmRB

### OBS Studio Crash Log URL

https://obsproject.com/logs/dBpOUBRk8NBw64aZ

### Expected Behavior

I wanted to move/set up my docks.

### Current Behavior

As soon as I try to connect a dock in the layout, like for example Twitch chat, OBS crashes.
More details:
- It still happens in safe mode
- If I run OBS as admin, the crashes stop (even with all my 30 plugins)
- Running a clean, new version of OBS as portable also solves it
- I tried re-running the installer. Did nothing

### Steps to Reproduce

1. Open OBS
2. Create or add a dock
3. Try to "dock it" into layout
4. Crash.
...

**Comment by RytoEX** [comment]
This is a crash deep in Qt's code. If I recall correctly, this is crashing in their RHI widgets path when reparenting a window. Unfortunately, I have never been able to reproduce this.

**Comment by rodduarte666** [comment]
As requested, here is a log where it happens in safe mode:
https://obsproject.com/logs/SiikdyRJuVsFUtEY

**Comment by RytoEX** [comment]
Is OBS running on your primary display at the time of the crash? If no, then can you try this on your primary display? If yes, then can you try this on a non-primary display?

**Comment by rodduarte666** [comment]
It's on a secondary. Lemme try on the main one...
Yup, still crashed.

**Comment by RytoEX** [comment]
Also, could you try moving docks around in this test app I've built and see if it crashes?
[minimal-qdockwidget-test.zip](https://github.com/user-attachments/files/16532017/minimal-qdockwidget-test.zip)

**Comment by rodduarte666** [comment]
So, no crashes, but the renameable dock won't dock inside the main app, not sure if this is intended :-)

**Comment by RytoEX** [comment]
Is this any kind of dock, or just browser docks?

**Comment by rodduarte666** [comment]
It's any docks... I just tried it with the AdvSS one.
The Twitch chat one also does it.
So I've replicated it with a plugin and a browser one.
And also a non-plugin one, like "scenes", while in safe mode.

**Comment by RytoEX** [comment]
Okay, so you can simply drag Scenes out of the window, drop it so it's floating, and then drag Scenes back into the window, drop it, and OBS crashes with just that?

**Comment by SuslikV** [comment]
Just in case, the crash is not v30.2.2 specific and happened before:
https://obsproject.com/forum/threads/obs-crashing-when-i-detach-the-live-chat-dock.176464/

**Comment by rodduarte666** [comment]
> Okay, so you can simply drag Scenes out of the window, drop it so it's floating, and then drag Scenes back into the window, drop it, and OBS crashes with just that?

Yep!

**Comment by SuslikV** [comment]
Does it crashes:
1. If you simply double-click the header of the docked pane and when it floats double-click the header again?
2. If you split your Scenes into different Scene Collections? (duplicate scene collections and delete unneeded scenes from each collection, so you have only about 100 sources+scenes in each scene collection, not 1000)

**Comment by rodduarte666** [comment]
Yep, the very first double click on any dock's title bar crashes it.
As for splitting the collection, I'm not gonna do that because there's no situation where I can actually use it that way.
But, I did test this by creating a brand new scene collection with NOTHING in it, and double-clicking a dock's title bar still crashed it.

**Comment by SuslikV** [comment]
I think you have some unusual data saved inside `global.ini` file (maybe file was damaged). You can access this file from main menu _File > Show Settings Folder_. You can open it with "notepad++" application and find the `[BasicWindow]` section. There are two parameters the `geometry` and the `DockState` in which common settings for the dock panes are stored. I think, if you copy this two strings of parameter values to new (portable) OBS it should start to crash on double-click too. This `global.ini` settings file stored separately from Profiles or Scene Collections but loads on application startup.

**Comment by RytoEX** [comment]
If you can verify the above test crashes a new portable mode OBS, could you please send me a copy of your `global.ini` file and current scene collection `.json` file? You can also send them if you do not know how to perform that test. If you prefer to send the files privately, I am RytoEX on the OBS Studio Discord server.

**Comment by rodduarte666** [comment]
[Desktop.zip](https://github.com/user-attachments/files/16617756/Desktop.zip)
Here you go! I don't mind, just please make this go away :-D

**Comment by notr1ch** [comment]
Try disabling the IBM Trusteer Rapport service.

**Comment by rodduarte666** [comment]
Tried it.
Nope, still crashes.

**Comment by rodduarte666** [comment]
Anything on this, guys?
